commit be3d217e813682e0be247ee03211e3abb13eb710 (refs/changes/30/3530/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-03-02T22:00:23+00:00 (1 year, 7 months ago)
    
    TRITON-198 - cloudapi test failures checking error message
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Approved by: Dave Eddy <dave.eddy@joyent.com>

diff --git a/test/machines.test.js b/test/machines.test.js
index 141cc58..e7b31c8 100644
--- a/test/machines.test.js
+++ b/test/machines.test.js
@@ -1088,7 +1088,7 @@ function (t) {
         t.ok(err, 'delete error expected');
         t.equal(res.statusCode, 409, 'http code');
         t.deepEqual(body, {
-            code: 'MachineUndeletableError',
+            code: 'CannotDestroyMachineError',
             message: 'Instance has "deletion_protection" enabled, ' +
                 'preventing deletion'
         }, 'check error message');
